plugin/acl/acl.go

```diff
@@ -28,53 +28,60 @@ type Rule struct {
 	Policies []Policy
 }
 
+// Action defines the action against queries.
+type Action int
+
 // Policy defines the ACL policy for DNS queries.
 // A policy performs the specified action (block/allow) on all DNS queries
 // matched by source IP or QTYPE.
 type Policy struct {
-	action int
+	action Action
 	qtypes map[uint16]struct{}
 	filter *iptree.Tree
 }
 
 const (
-	// Allow allows authorized queries to recurse.
-	Allow = iota
-	// Block blocks unauthorized queries towards protected DNS zones.
-	Block
+	// ActionNone does nothing on the queries.
+	ActionNone = iota
+	// ActionAllow allows authorized queries to recurse.
+	ActionAllow
+	// ActionBlock blocks unauthorized queries towards protected DNS zones.
+	ActionBlock
 )
 
 func (a acl) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg) (int, error) {
 	state := request.Request{W: w, Req: r}
+Check:
```

Comment:
Can it be smth more meaningful, e.g. `rulesLoop:`

Reply:
Sure